crates/weaver_semconv/src/group.rs

```diff
@@ -114,11 +118,22 @@ impl GroupSpec {
         }
 
         // Field name is required if prefix is empty and if type is event.
-        if self.r#type == GroupType::Event && self.prefix.is_empty() && self.name.is_none() {
+        if self.r#type == GroupType::Event {
+            if self.prefix.is_empty() && self.name.is_none() {
```

Comment:
Is it a `&&` or a `||` condition between the presence of a prefix and the presence of a name?

Reply:
Good question let me review all of the existing `event` types in the semantic registry to see if they have names already. But for log based events the name is required and the prefix is not (as that is only applied to attributes (not the body fields).